backends/audiocd/default/default-audiocd.cpp

```diff
@@ -134,6 +135,27 @@ bool DefaultAudioCDManager::play(int track, int numLoops, int startFrame, int du
 	return false;
 }
 
+bool DefaultAudioCDManager::playAbsolute(int startFrame, int numLoops, int duration, bool onlyEmulate,
```

Comment:
I would recommend adding an optional variable with the default value `"disc.cue"` here instead of hardcoding it like this.

Reply:
Done. I don't quite know the C++ semantics, was `const char*` as the type fine?

Reply:
Yes, just the code formatting is off

Reply:
Done!